libs/ledger-live-common/src/e2e/enum/AppInfos.ts

```diff
@@ -47,7 +47,7 @@ export class AppInfos {
 
   static readonly NEAR = new AppInfos("Near");
 
-  static readonly MULTIVERSE_X = new AppInfos("Multiverse X");
+  static readonly MULTIVERS_X = new AppInfos("Multiverse X");
```

Comment:
`new AppInfos("Multiverse X") ` isn't `new AppInfos("Multivers X")`?

Reply:
Fix it, thanks